Src/CSharpier.Cli/FormattingCache.cs

```diff
@@ -0,0 +1,112 @@
+using System.Collections.Concurrent;
+using System.IO.Abstractions;
+using Newtonsoft.Json;
```

Comment:
I know this is just a POC, but you should avoid adding new dependencies on `Newtonsoft`.
Use `System.Text.Json` instead.